config/environments/test.rb

```diff
@@ -31,7 +31,7 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
-  config.time_zone = ENV.fetch("TIMEZONE", "Melbourne")
+  config.time_zone = "UTC"
```

Comment:
Good! I was fearing this would break lots of tests but it turns out we were lucky.

Reply:
Indeed, removing this from `config/application.rb` and here leaves `config.time_zone` leaves this as `UTC`. We could perhaps comment out `TIMEZONE` from `config/application.yml.example`.

Reply:
But don't we want it to be configured for non-test environments?

Reply:
yes, those can be configured in `config/environment/production.rb` and `config/environment/staging.rb`.